web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -11,31 +9,18 @@
     padding: $unit * 3;
 
     .card-labels {
-      display: flex;
-      align-content: center;
-      text-transform: uppercase;
-      color: $primary;
-      justify-content: space-between;
 
-      .field--name-field-r-cooking-time {
+      .inline-field-label {
         background-color: $white;
-        font-size: $xsmall-font;
-        text-align: center;
-        padding: $unit;
 
-        .field__item {
-          display: block;
-        }
-      }
+        .field__label,
+        .field__value {
+          font-size: 10px;
 
-      .field--name-field-r-serves {
-        display: flex;
-        align-items: center;
-        background-color: $white;
-        font-size: $xsmall-font;
-        text-align: center;
-        margin-right: 0;
-        padding: $unit;
+          @include breakpoint($desk-wide) {
+            font-size: $xsmall-font;
+          }
+        }
```

Comment:
You should not be decreasing the font size here and the bug is still there

Reply:
The "bug" only appears when an excessive amount of text is added into the field. The font size decrease is intentional

Reply:
Your system should be built in a way that prevents bugs like that regardless of the amount of text and also the font size decrease is inaccessible because it's too small. https://accessibility.psu.edu/fontsizehtml/

Reply:
agreed, thank you. It's done.